Cargo.toml

```diff
@@ -17,11 +17,13 @@ members = [
     # Domain
     "src/domain/opendatafabric",
     "src/domain/accounts/domain",
+    "src/domain/dataset-env-vars/domain",
```

Comment:
Maybe a bit too narrow name for a domain. Sooner or later we will have dataset entity present in the database, and other connected data, like permissions for a private datasets. Other features would follow. I would reserve a wider name, maybe `src/domain/dataset`